core/commands/add.go

```diff
@@ -266,7 +266,7 @@ remains to be implemented.
 					if quiet {
 						fmt.Fprintf(res.Stdout(), "%s\n", output.Hash)
 					} else {
-						fmt.Fprintf(res.Stdout(), "added %s %s\n", output.Hash, output.Name)
+						fmt.Fprintf(res.Stdout(), "added %s %s\nhttps://ipfs.io/ipfs/%s\n", output.Hash, output.Name, output.Hash)
```

Comment:
sorry, this is an incompatible change. Build a tool on top of `ipfs add` to do this please. see how i do it in: https://github.com/jbenet/ipfs-screencap/blob/master/ipfs-screencap#L88-L94